### PR TITLE
[Merged by Bors] - feat(MeasureTheory): aesop rules for strong measurability + `measurability?` tactic

### PR DESCRIPTION
**Pull request by dupuisf**
This PR adds aesop tags to a few lemmas pertaining to strong measurability, allowing to prove e.g. `StronglyMeasurable Real.log` using the `measurability` tactic.

It also implements `measurability?` via `aesop?`.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by sgouezel** [comment]
Can you add some tests to `test.measurability`? Also, many commented tests in this file compile just fine now, it is maybe time to uncomment them.

**Comment by dupuisf** [comment]
I went down a bit of a rabbit hole trying to get all the tests to work, but now it looks OK. In addition to tagging a lot more lemmas, I fiddled with aesop penalties on the various tagged lemmas to get the tactic to try lemmas in the right order. The main change is that lemmas such as
```
theorem Measurable.mul [MeasurableMul₂ M] (hf : Measurable f) (hg : Measurable g) :
    Measurable fun a => f a * g a
```
are now tagged with `@[aesop safe 20 apply (rule_sets [Measurable])]`, which downgrades its priority compared with `const_mul` and `mul_const` which have the default penalty of 1. I also downgraded the priority of composition lemmas and other lemmas that caused problems.

File: test/measurability.lean [hunk 1]
```diff
@@ -59,7 +82,11 @@ example [Div β] [MeasurableDiv₂ β] (hf : Measurable f) (hg : Measurable g)
     (ht : MeasurableSet t₂): MeasurableSet ((fun x => f x / g x) ⁻¹' t₂) := by measurability
```
Comment (sgouezel) [inline test/measurability.lean:82]:
```suggestion
    (ht : MeasurableSet t₂) : MeasurableSet ((fun x => f x / g x) ⁻¹' t₂) := by measurability
```

**Review by sgouezel — COMMENTED** [review]
bors d+
Thanks!

**Pull request closed without merging**